linode/manager
# M3-5898: Update `marketplace` with new marketplace apps

## Description 📝

- Updates new marketplace apps (from #8476) to match new `marketplace` changes 
  - Budibase
  - HashiCorp Nomad
  - HashiCorp Vault
- This will fix tsc errors in the `marketplace` branch

## How to test 🧪

- Check Icons and Colors for the following marketplace apps at `http://localhost:3000/linodes/create?type=One-Click`
  - Budibase
  - HashiCorp Nomad
  - HashiCorp Vault
